src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPanelMapping.java

```diff
@@ -40,17 +45,24 @@ private HearingsPanelMapping() {
     public static PanelRequirements getPanelRequirements(SscsCaseData caseData,
                                                          ReferenceDataServiceHolder referenceDataServiceHolder) {
         return PanelRequirements.builder()
-            .roleTypes(getRoleTypes())
+            .roleTypes(getRoleTypes(caseData.getBenefitCode()))
             .authorisationTypes(getAuthorisationTypes())
             .authorisationSubTypes(getAuthorisationSubTypes())
             .panelPreferences(getPanelPreferences(caseData))
             .panelSpecialisms(getPanelSpecialisms(caseData, getSessionCaseCode(caseData, referenceDataServiceHolder)))
             .build();
     }
 
-    public static List<String> getRoleTypes() {
-        //TODO Need to retrieve RoleTypes from caseData and/or ReferenceData
-        return Collections.emptyList();
+    public static List<String> getRoleTypes(String benefitCode) {
+        List<String> roleTypes = new ArrayList<>();
+        if (isNotBlank(benefitCode)) {
```

Comment:
I don't think we should be implementing things like this. If we're asked to add more cases there will be more and more if blocks. I'd recommend something like creating an enum that we can search based on the benefit code. This enum would also have a consumer which takes in List<String> (roletypes). You'd find the corresponding enum then invoke its consumer which should add the correct reference to the list.

Reply:
Added a new ENUM as BenefitRoleRelationType and retrieved role types by using it' s static method as below:

public static List<String> getRoleTypes(String benefitCode) {
        return BenefitRoleRelationType.findRoleTypesByBenefitCode(benefitCode);
    }